README.md

```diff
@@ -43,37 +43,51 @@ tota11y uses a variety of technologies, including [jQuery](https://jquery.com/),
 
 You can run unit tests on tota11y with the following:
 
-```
+```bash
 npm test
 ```
 
 Or lint with:
 
-```
+```bash
 npm run lint
 ```
 
 To perform manual testing as you work, you can run a live dev-server with the
 following:
 
-```
+```bash
 npm start
 ```
 
 ## Building
 
 To create a development build as the test server uses:
 
-```
+```bash
 npm run build:dev
 ```
 
 To create a production build, with minified and unminified output:
 
-```
+```bash
 npm run build:prod
 ```
 
+## Releasing
+
+Currently, the following steps must be made to release a new version of tota11y:
+
+1. Update `package.json` with the version number to be released.
+1. Commit the release details to the CHANGELOG.md.
```

Comment:
It looks a little weird seeing all `1.`s, but the rich diff checks out.

Reply:
I wonder if this should be a PR instead of a commit?

Reply:
The  `1.`s are valid Markdown. This way, you have to update less in case the order changes.
However, counting them up should work also …

Reply:
@kevinbarabash Good point on the wording about committing the change. It should definitely be a PR. I'll update.